cloudwatch/cloudwatch.go

```diff
@@ -97,6 +97,11 @@ type TaskMetadata struct {
 	TaskID  string `json:"TaskID,omitempty"`
 }
 
+type StreamDoesntExistError struct {
```

Comment:
please resolve this check failure: `exported type StreamDoesntExistError should have comment or be unexported`